specification/cognitiveservices/Language.AnalyzeText/models/common.models.tsp

```diff
@@ -190,6 +190,10 @@ model AnalyzeTextJobsInput {
   /** Default language to use for records requesting automatic language detection. */
   @added(Versions.v2024_11_15_preview)
   defaultLanguage?: string;
+
+  /** Optional duration in seconds after which the job will be canceled if not completed. */
+  @added(Versions.v2025_05_15_preview)
+  cancelAfter?: float32;
```

Comment:
This violates REST API guidelines. Units of time measurement should be included in the name e.g., `cancelAfterSeconds`. But why a float? Floats aren't very portable across languages. If you need fractions of a second, use milliseconds e.g., `cancelAfterMs`.